### ISSUE TITLE
Tray behavior on Windows refinement

### ISSUE THREAD

**Issue by NoahAndrews** [heyman/heynote#141]
Posting as a follow-up to https://github.com/heyman/heynote/issues/137 (thanks for working to iron this out before releasing the stable version, you've been extremely responsive, and I really appreciate that, without expecting it as the norm).

### Improvements in 1.6 beta 2:
* Closing the heynote window keeps the tray icon present, allowing for easier access without having to keep heynote pinned in the taskbar

### Regressions in 1.6 beta 2:
* heynote is no longer displayed in the alt-tab switcher, presumably because it's hidden from the taskbar

### Continued suboptimal tray icon behavior from beta 1
* The global hotkey does not work when heynote's window is closed

**Comment by NoahAndrews** [comment]
This may be related to #140

**Comment by heyman** [comment]
Thanks for the report. I *think* #142 should fix it. However, there are subtle differences between how the Electron Tray API works in Windows (Tray icons) and on Mac (Menu bar), and the feature also interact with the global hotkey feature (which also works a differently on both OS:s), so there's definitely room for more bugs. At least I've tested it on both Mac and Windows now. However, I have not tested the new changes is Linux, so fingers crossed it behaves well there 🤞

I've released a new beta version (1.6.0-beta.3) that has the fix.

One other thing that I also fixed is that auto-updating from 1.6.0-beta.2 will not work if Tray/Menu bar setting is turned on.
